app/components/PinEditButton.js

```diff
@@ -0,0 +1,60 @@
+import React, { Component, AlertIOS , View, StyleSheet} from 'react-native';
+import Button from 'react-native-button';
+
+export default class PinEditButton extends Component{
+  constructor(props) {
+    super(props);
+  }
+
+  editTitle(value) {
+    const { pin, updatePins, updateRecent } = this.props;
+    
+    updatePins(pin, value);
+    updateRecent();
+  }
+
+  render() {
+    const { pin, deletePin, hideButton } = this.props;
+    return(
+      <Button
+        style={[styles.bubble, styles.button]}
+
+        onPress={ () => {
+          hideButton();
+          AlertIOS.prompt(
+            pin.title,
+            'Editting Pin',
```

Comment:
We can probably refine the language of the pop-up (for one thing, "Editing" has one "t" :stuck_out_tongue_closed_eyes:), but that's a minor thing and doesn't interfere with the functionality of this PR.